Update Swiss Hydrological Data documentation

Bouni opened the pull request:
**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17166

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

Bouni committed:
Update Swiss Hydrological Data documentation

WofWca commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -13,15 +13,15 @@ ha_iot_class: "Cloud Polling"
 
-<p class='note warning'>
-  This sensor doesn't work at the moment due to changed by the [Swiss Federal Office for the Environment (Bundesamt für Umwelt - Abt. Hydrologie)](http://www.hydrodaten.admin.ch) to access the data.
+<p class='note info'>
```
IIRC `class='note'` is enough.

WofWca commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -44,24 +46,46 @@ name:
+  - The mean discharge measurement of the last 24 hours
 
-This sensor contains additional information which an easily accessed by a [template sensor](/components/sensor.template/).
+<p class='note info'>
```
Same

frenck commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -44,24 +46,46 @@ name:
+  description: list of measurements you want to use.
+  required: true
+  type: list
+  deafult:
```
Typo: `default`. Also, default should have a value.

frenck commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -44,24 +46,46 @@ name:
+  required: true
+  type: list
+  deafult:
 {% endconfiguration %}
```
This is a list and that should have a `keys` attribute. You can use that attribute to list the possible measurement values.

frenck commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -44,24 +46,46 @@ name:
+Valid measurement values are:
 
-## {% linkable_title Example %}
+- temperature
```
Move this section into the `keys` second of the `measurements` configuration value.

frenck commented on source/_components/sensor.swiss_hydrological_data.markdown:
```diff
@@ -2,7 +2,7 @@
 title: "Swiss Hydrological Data"
 description: "Instructions on how to integrate hydrological data of Swiss waters within Home Assistant."
-date: 2016-06-17 17:00
+date: 2018-10-05 12:00
```
Please do not update the date. It marks the initial introduction.

frenck reviewed (dismissed):
@Bouni Thank you for this PR. Some comments were made on it. Could you please take a look? 👍

Bouni commented:
I've got some feedback on the component itself. Therefore I will update fix the documentation as soon as I've found a solution for the addressed issues with the component!

frenck commented:
Thanks for letting us know @Bouni!
We'll wait patiently 👍

frenck commented:
Hold on sec, this PR depends on a parent PR. 
This docs PR should go into the `next` branch.
@Bouni Please be sure to update the target and rebase your changes.

Bouni commented:
@frenck I've used **current** because the documentation says that updates of existing components should go into **current**, just documentation on new components should go into **next**.
Anyway, I'll do what you said :+1:

frenck commented:
@Bouni I understand the confusion.

Updates to the current release or documentation fixes go into the `current` branch. Changes that depend on a next release go into `next`. Since you are changing code (which results in changing docs), the change should go into `next`. That documentation will go live when the next release takes place.

This ensures the docs & release are in sync (and matches).

fabaff committed:
Update

fabaff committed:
Fix typo and style

fabaff committed:
Fix my copy-&-paste mistake:

fabaff reviewed (approved):
:bird:
